.github/workflows/build.yml

```
@@ -51,12 +51,18 @@ jobs:
           - php: 8.2
             os: ubuntu-latest
             composer-mode: update
-            symfony-version: '5.4'
+            symfony-version: '5.4.*'
 
           - php: 8.2
             os: ubuntu-latest
             composer-mode: update
-            symfony-version: '6.4'
+            symfony-version: '6.4.*'
+
+          - php: 8.4
+            os: ubuntu-latest
+            composer-mode: update
+            composer-minimum-stability: beta
+            symfony-version: '^8.0.0-BETA-1'
```

I think with the minimum-stability set to beta, we don't need to explicitly specify it in the version constraint as well - if I install locally on this branch with SYMFONY_REQUIRE=8.0.* I'm getting the same packages as the actions run installed here.

```suggestion
            symfony-version: '8.0.*'
```